frontend/src/TooljetDatabase/Forms/TableSchema.jsx

```diff
@@ -0,0 +1,332 @@
+import React from 'react';
+import { UniqueConstraintPopOver } from '../Table/ActionsPopover/UniqueConstraintPopOver';
+import { ToolTip } from '@/_components/ToolTip';
+import Serial from '../Icons/Serial.svg';
+import ForeignKeyRelation from '../Icons/Fk-relation.svg';
+import IndeterminateCheckbox from '@/_ui/IndeterminateCheckbox';
+import SelectIcon from '../Icons/Select-column.svg';
+import MenuIcon from '../Icons/Unique-menu.svg';
+// import DeleteIcon from '../Icons/DeleteIcon.svg';
+import Tick from '../Icons/Tick.svg';
+import tjdbDropdownStyles, { dataTypes, formatOptionLabel, serialDataType } from '../constants';
+import Select, { components } from 'react-select';
+
+function TableSchema({ columns, setColumns, darkMode, columnSelection, setColumnSelection, handleDelete, isEditMode }) {
+  const { Option } = components;
+
+  const darkDisabledBackground = '#1f2936';
+  const lightDisabledBackground = '#f4f6fa';
+  const lightFocussedBackground = '#fff';
+  const darkFocussedBackground = 'transparent';
+  const lightBackground = '#fff';
+  const darkBackground = 'transparent';
+
+  const darkBorderHover = '#dadcde';
+  const lightBorderHover = '#dadcde';
+
+  const darkDisabledBorder = '#3a3f42';
+  const lightDisabledBorder = '#dadcde';
+  const lightFocussedBorder = '#90B5E2 !important';
+  const darkFocussedBorder = '#90b5e2 !important';
+  const lightBorder = '#dadcde';
+  const darkBorder = '#dadcde';
+  const dropdownContainerWidth = '360px';
+
+  const CustomSelectOption = (props) => {
+    const isCheckDataType =
+      columns[columnSelection.index]?.column_default?.includes('nextval(') &&
```

Comment:
can we also add `::regclass` being present in the default value as well?

We can use a regex like: `/^nextval\(".*"::regclass\)$/`